CIFAR10/train/diux_cifar_g3.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "v1.0",
+    "author": "DIUX",
+    "authorEmail": "ybulatov.ctr@diux.mil",
+    "framework": "tensorflow 1.5, tensorpack 0.8.1",
+    "codeURL": "https://github.com/ppwwyyxx/tensorpack/blob/master/examples/ResNet/cifar10-resnet.py",
+    "model": "ResNet50",
+    "hardware": "g3.4xlarge",
+    "timestamp": "2018-01-31",
+    "costPerHour": 1.14,
+    "optimizer": "SGD with Momentum",
+    "momentum": 0.9,
+    "schedule": [
```

Comment:
Can you move schedule into the misc object:

```
{
	"version": "v1.0",
	"author": "DIUX",
	"authorEmail": "ybulatov.ctr@diux.mil",
	"framework": "tensorflow 1.5, tensorpack 0.8.1",
	"codeURL": "https://github.com/ppwwyyxx/tensorpack/blob/master/examples/ResNet/cifar10-resnet.py",
	"model": "ResNet50",
	"hardware": "g3.4xlarge",
	"timestamp": "2018-01-31",
	"costPerHour": 1.14,
	"optimizer": "SGD with Momentum",
	"momentum": 0.9,
	"misc": {
		"AMI": "ami-9ba7c4e1",
		"schedule": [{
				"learning_rate": 0.1,
				"epochs": 82
			},
			{
				"learning_rate": 0.01,
				"epochs": 41
			},
			{
				"learning_rate": 0.001,
				"epochs": 177
			},
			{
				"learning_rate": 0.0002,
				"epochs": 100
			}
		]
	}
}
```

Note this includes the correction to my comment below